server/wfnews-persistence/src/main/java/ca/bc/gov/nrs/wfnews/persistence/v1/dao/mybatis/PublishedIncidentDaoImpl.java

```diff
@@ -225,7 +225,7 @@ public PagedDtos<PublishedIncidentDto> select(String searchText, Integer pageNum
 			parameters.put("fireOfNote", fireOfNote);
 			parameters.put("stageOfControlList", stageOfControlList);
 			parameters.put("newFires", newFires);
-			parameters.put("fireCentre", fireCentre);
+			parameters.put("fireCentreCode", fireCentreCode);
```

Comment:
@yzlucas @dhlevi as mentioned, maybe fireCentreName should be added here for filtering.

Reply:
thanks, @ssylver93  
just added filtering for fireCentreName